modules/router-certificates.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/certificate-reference.adoc
+
+[id="router-certificates_{context}"]
+= Router certificates
+
+[discrete]
+== Purpose
+
+The Ingress Operator uses certificates for:
+
+* Securing access to metrics for Prometheus.
+* Securing access to Routes.
+
+[discrete]
+== File path
+
+To secure access to Ingress Operator and Ingress Controller metrics, the Ingress
+Operator uses service serving certificates. The Operator requests a certificate
+from the service-ca controller for its own metrics, and the service-ca
+controller puts the certificate in a secret named metrics-tls in the
+openshift-ingress-operator namespace. Additionally, the Ingress Operator
+requests a certificate for each Ingress Controller, and the service-ca
+controller puts the certificate in a secret named `router-metrics-certs-<name>`,
+where `<name>` is the name of the Ingress Controller, in the
+`openshift-ingress`. namespace.
+
+Each Ingress Controller has a default certificate that the Controller uses for
+secured routes that do not specify their own certificates. You can set a custom
+default certificate, or else the Operator generates a certificate for you. The
+Operator uses its own self-signed signing certificate to sign any default
+certificate that it generates. The Operator generates this signing certificate
+and puts it in a secret named router-ca in the openshift-ingress-operator
+namespace. When the Operator generates a default certificate, it puts the
+default certificate in a secret named router-certs-<name> (where <name> is the
+name of the Ingress Controller) in the openshift-ingress namespace.
+
+[WARNING]
+====
+The Operator generates a default certificate for an Ingress Controller to serve
+as a placeholder till you configure a custom default certificate. Do not use
+operator-generated default certificates in production clusters.
+====
+
+[discrete]
+== Expiration
+
+The expiration terms for the Ingress Operator's certificates are as follow:
+
+* The expiration date for metrics certificates that the `service-ca` controller
+creates is two years after the date of creation.
+* The expiration date for the Operator's signing certificate is two years after
+the date of creation.
+* The expiration date for default certificates that the Operator generates is two
+years after the date of creation.
+
+You cannot specify custom expiration terms on certificates that the Ingress
+Operator or `service-ca` controller creates.
```

Comment:
Whoops!  My fault: "creates" → "create".  (Alternatively, if the proximity rule applies, then it should apply in the following paragraph, in which case we need to replace "create" → "creates" in the sentence below.)

Reply:
@Miciah Nice catch. Thanks!